### ISSUE TITLE
Module Re-Ordering Thoughts

### ISSUE THREAD

**Issue by njlyon0** [lter/ssecr#43]
## Summary

We may want to consider re-ordering some of the modules as we saw how they were useful to the Synthesis Fellows groups.

## Sub-Tasks

- [x] Keep an eye on module order & feedback throughout the course

## Resources

- TBD

**Comment by njlyon0** [comment]
## Reproducibility vs. Data Discovery

Could make Data Discovery be the 'during the in-person week' module and put Reproducibility after so that groups know how to find data as early as possible and may have early scripts to make more reproducible during that first virtual module

**Comment by njlyon0** [comment]
## Data Visualization vs. Data Wrangling

Feels like a semi-arbitrary division. To make certain kinds of viz you need some wrangling tools that we don't have time to cover in the modules as they are. Likely will make sense to refactor these as "working with data 1" and "working with data 2"

### TLDR: consider refactoring Data Viz & Data Wrangling by _when_ groups need those skills

**Comment by ckappel** [comment]
### Facilitation
Students seemed to need the tools for convergence / group decisionmaking early in the course, but some of these come later in DEC thinking.

**Comment by marty-downs** [comment]
Move data discovery before in person week. Spend more time/effort helping them think about/develop project ideas before in-person week. Revisit draft questions before in-person week.
